### PR TITLE
[CON-3] Prepare Console for Vercel deployments

### PR DESCRIPTION
**Pull request by awmuncy**
This PR does two things:

1. It disables the action which pushes the latest version of the Console to IPFS on merge to main. 
2. Changes the directory to which the final files for the console are compiled to be more conventional for it's Vercel deployment.

### CODE REVIEW

**Comment by joewagner** [comment]
@awmuncy I put a few questions inline.  I also noticed that the console app has quite a few errors in the "dev console" (no pun intended).  I don't think they have anything to do with this PR, but if there's not already work scoped to fix those we should probably capture that somewhere.

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
 on:
-  push:
-    branches: [ main ]
-  pull_request:
-    branches: [ main ]
+  # push:
+  #   branches: [ main ]
+  # pull_request:
+  #   branches: [ main ]
```
[the remark this answers is not in the document]
Comment (awmuncy) [reply .github/workflows/node.js.yml:7]: This was part of disabling the IPFS deploy, but I'll remove the file entirely, as it is no longer needed.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 /node_modules
+/public
 /dist
```
[the remark this answers is not in the document]
Comment (awmuncy) [reply .gitignore:3]: Nope, this is an artifact, I'll remove it.

**Comment by awmuncy** [comment]
> @awmuncy I put a few questions inline. I also noticed that the console app has quite a few errors in the "dev console" (no pun intended). I don't think they have anything to do with this PR, but if there's not already work scoped to fix those we should probably capture that somewhere.

I added some errors to the console after the network upgrade a few months ago. Certain tables which were previously valid tables are no longer processed by the validator. For example, any table that used the `any` type. Those are no longer picked up when querying the validator for metadata. I threw an error in those cases so it didn't completely ignore tables someone owned with _no_ information on why. Those errors aren't related to this change, but I can add a ticket to the CON team in linear to deal with them.

**Comment by joewagner** [comment]
> I added some errors to the console after the network upgrade a few months ago. Certain tables which were previously valid tables are no longer processed by the validator. For example, any table that used the `any` type. Those are no longer picked up when querying the validator for metadata. I threw an error in those cases so it didn't completely ignore tables someone owned with _no_ information on why. Those errors aren't related to this change, but I can add a ticket to the CON team in linear to deal with them.

Sounds good.  We can discuss more in the linear issues if/when that makes sense.

**Review by joewagner — APPROVED** [review]
LGTM!

**Pull request merged**